src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature

```diff
@@ -0,0 +1,44 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Validate Creation for Absence Event Category Descriptors
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+
+        @ignore
+        Scenario: Verify new resource can be created successfully
+             When sending a POST request with valid body to
+                  | Name          | Endpoint                               |
+                  | academicWeeks | /ed-fi/absenceEventCategoryDescriptors |
```

Comment:
The C# developer who picks up this story will have to decide about the payload. In this case perhaps you can put the exact payload in here.

"Name = academicWeeks" is actually irrelevant to this descriptor endpoint.